### DISCUSSION TITLE
Vectorized forecast for models that could support it

### DISCUSSION THREAD

**Discussion by albertcthomas** [Nixtla/statsforecast#923] [Q&A]
Hi, If I have a model like a tabular model (sklearn model for instance) and a test set that is made of multiple slices of the time series, each slice corresponding to an input (x_(t-L+1), …, x_(t)). So the test set has multiple values for t (a lot) and I want to apply the fixed model that was trained on a training set. Is it possible with the current API of predict/forecast to pass a batch of such inputs and get the predictions without having to implement the for loop ? With the tabular approach this would be possible, benefitting from the vectorized prediction instead of a for loop or the need for parallelization.
